dataclients/kubernetes/kube.go

```diff
@@ -23,7 +24,7 @@ const (
 	serviceHostEnvVar            = "KUBERNETES_SERVICE_HOST"
 	servicePortEnvVar            = "KUBERNETES_SERVICE_PORT"
 	httpRedirectRouteID          = "kube__redirect"
-	defaultLoadBalancerAlgorithm = "roundRobin"
+	DefaultLoadBalancerAlgorithm = "roundRobin"
```

Comment:
Since it is exported now I think its better to move it into separate cont block because all others are not exported.